Doctrine/RouteConditionMetadataListener.php

```diff
@@ -49,6 +49,11 @@ public function loadClassMetadata(LoadClassMetadataEventArgs $eventArgs)
         }
 
         $meta = $eventArgs->getClassMetadata();
+
+        if (null === $meta->getReflectionClass()) {
```

Comment:
is this something that is supposed to ever happen? we would be loading metadata, but not about a class? or in what situation does this problem occur?